tests/test_taxi_service.py

```diff
@@ -122,8 +118,6 @@ def setUp(self):
         self.credentials = {"username": "admin.user", "password": "1qazcde3"}
 
     def test_login(self):
-        response = self.client.post(
-            reverse("login"), self.credentials, follow=True
-        )
+        response = self.client.post(reverse("login"), self.credentials, follow=True)
         self.assertTrue(response.context["user"].is_active)
         print(response.context["user"])
```

Comment:
Remove the `print` statement from the `LogInTest` class to keep the test output clean and professional. Debugging statements should not be present in the final test code.